davidbrucetn/nss-capstone-frontend-dwb
# Fixed auth check for new user reg userid

# Description
Revision of authorization checks after new user registered. 

Fixes # (issue)
## Type of change
Please delete options that are not relevant.
       Now pulls email from session/local storage and pulls user id from API with that email. Before this, when a new user registered and started using the program, there was no id in the local/session storage, so the items were not saved.

# Testing Instructions
Create a new user and start using the program (without logging out and back in). Search restaurants and save to your collection. Verify that items show up after adding.

# Checklist:
